docs/plugins/known-plugins.asciidoc

```diff
@@ -47,5 +47,6 @@ This list of plugins is not guaranteed to work on your version of Kibana. Instea
 [float]
 === Other
 * https://github.com/nreese/kibana-time-plugin[Time picker as a dashboard panel] Widget to view and edit the time range from within dashboards.
+* https://github.com/Webiks/kibana-API.git[Kibana-API] (webiks) Expose kibana functionality to your application, like create visualization dynamiclly, delete visualization and more..
 
```

Comment:
Could you tidy up the language a little bit? 

e.g. :`Exposes an API with Kibana functionality. Use it to create, edit and embed visualizations.`